example/cats.py

```diff
@@ -79,7 +66,17 @@ class CatStats(Coverpoint):
     specifying values to the axis.
     """
 
-    def __init__(self, name: str, description: str):
+    NAME = "cat_stats"
+    DESCRIPTION = "Some basic cat stats"
+    TIER = 1
+    TAGS = ["basic", "stats"]
+
+    def __init__(
+        self,
+        name: str | None = None,
+        description: str | None = None,
+        motivation: str | None = None,
+    ):
```

Comment:
There is a lot more boilerplate now - I know we discussed beforehand, but I kind of hate it

In this case at least, I think you would write:
```python
    def __init__(
        self,
        name: str = "cat_stats",
        description: str = "Some basic cat stats",
        motivation: str | None = None,
    ):
```

Reply:
I hadn't added defaults to init, so the user didn't have to override it if no additional arguments were passed in other than name, description or motivation